fiaas_deploy_daemon/config.py

```diff
@@ -125,6 +125,10 @@
 if the application doesn't specify any.
 """
 
+ENABLE_SERVICE_ACCOUNT_PER_APP = """
+Look for a service account by app name; use this if we find one, else fall back to
+the namespace's default.
+"""
```

Comment:
I don't think this text matches the behavior as it works in this branch. Maybe it should be changed to something like "Use a service account matching the application name".